ods_ci/libs/Helpers.py

```diff
@@ -167,12 +176,7 @@ def uninstall_managed_starburst_using_addon_flow(self, cluster_name):
     @keyword
     def inference_comparison(self, expected, received, threshold=0.00001):
         try:
-            import ast
-            import decimal
-            import numbers
-            import re
```

Comment:
Why did this move the imports out of the function?
Usually when one of these keywords is called it's a one off and no other keywords get called, so IMHO it's more efficient to only import inside the function so that it's triggered only when actually needed

Reply:
The reason behind it is PEP8 recommendation that are used as part of pylint - https://peps.python.org/pep-0008/#imports . My personal preference is to have all imports and the top level. I am not sure if this somehow influence performance in the bad way to see the real difference, but if you want to keep it on function level, I can revert it and remove the check for it.

Reply:
we're talking about microseconds so it's probably not a big deal, but yeah there is a performance impact to importing libraries that do not get used. I'm not sure how Robot Framework behaves with this helper file specifically, I had put them inside the function under the assumption that the import would happen only if the keyword gets called during execution. We can leave it as is if PEP8 recommends top level imports.

Reply:
I think it works exactly as you describe. We can wait for some other reviews and thoughts about it.